packages/suite-desktop/src-electron/mocks/electron-updater.ts

```diff
@@ -58,7 +58,7 @@ const updateInfo = {
     ],
     path: 'Trezor-Suite-31.33.7-fake.AppImage',
     sha512: 'tfvdNXsjMe8YXJwTuujz4tKTdfsCuR/9VECF8EkcRP95YM7vuDV8dumru1jKtdiv0gaS1GT3SPEeAfmczY5jGg==',
-    releaseDate: '2029-14-12T13:39:03.416Z',
+    releaseDate: '2029-12-12T13:39:03.416Z',
```

Comment:
i guess this downgrade is for test purposes?

Reply:
There was an invalid date. Fourteenth month.

Reply:
Resolving